pkg/installer/generateconfig.go

```diff
@@ -222,6 +222,7 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 						Architecture:   types.ArchitectureAMD64,
 					},
 				},
+				CredentialsMode: determineCredentialsMode(m.oc.Properties.PlatformWorkloadIdentityProfile),
```

Comment:
We are now defining the CredentialsMode to be  `PassthroughCredentialsMode` in case of classic cluster, which is default and that's why it was not defined earlier.
What do you think if we only set the CredentialsMode field only when the cluster is MIWI?

Reply:
I have no objection to that